locals.tf

```diff
@@ -7,6 +7,8 @@ locals {
   resource_group                        = local.existing_resource_group == "" ? azurerm_resource_group.default[0] : data.azurerm_resource_group.existing_resource_group[0]
   enable_resource_group_lock            = var.enable_resource_group_lock
   key_vault_access_users                = toset(var.key_vault_access_users)
+  key_vault_access_ipv4                 = toset(var.key_vault_access_ipv4)
+  key_vault_access_subnet_ids           = toset(var.key_vault_access_subnet_ids)
```

Comment:
I don't think we need to convert them to a set here
It would be useful for Azure to throw an error if there are duplicates provided in tfvars